tutorials/examples/cov2d_experiment

```diff
@@ -0,0 +1,52 @@
+"""
+This script illustrates denoising 2D images using batched Cov2D class
+from experime dataset and outputing to mrcs file.
+"""
+
+import logging
+
+from aspire.basis.ffb_2d import FFBBasis2D
+from aspire.denoising.denoiser_cov2d import DenoiserCov2D
+from aspire.estimation.noise import AnisotropicNoiseEstimator
+from aspire.source.relion import RelionSource
+
+logger = logging.getLogger('aspire')
+
+# Set input path and files and initialize other parameters
+data_folder = '/tigress/junchaox/CryoEMdata/empiar10028'
```

Comment:
The convention I've always seen is to specify paths that need to be modified before the script can be expected to run in ALL_CAPS. That way they stand out. Also if `empiar10028` can be reconstructed by any user by following a specific set of steps, then you may want to specify these in comments, and/or as the default path (e.g. `/path/to/untarred/empiar/dataset`) etc.

Reply:
Done in new commit.